code/modules/mob/living/silicon/ai/ai.dm

```diff
@@ -435,6 +438,7 @@ var/list/ai_verbs_default = list(
 /mob/living/silicon/ai/emp_act(severity)
 	if (prob(30))
 		view_core()
+	icon_state = "ai-fuzz"

```

Comment:
does this fuzzy icon state ever get undone?

iirc, EMPing an AI causes damage but its not necessarily permanant.